feature-utils/poly-import/test/dataAccount.test.js

```diff
@@ -0,0 +1,22 @@
+import { DataAccount } from "../src";
+
+describe("DataAccount has the correct attributes", () => {
+    let dataAccount = new DataAccount();
+    it("Attributes have the correct type", () => {
```

Comment:
I think we should get rid of this one - unit tests to try to ensure type safety does not seem like a good idea to me. It's quite easy to forget to extend this test when adding additional attributes, and it won't catch cases where the type changes unexpectedly, giving a false sense of security. It's just dynamically typed, something we need to accept. Tests should attempt to break the logic, not verify weird details about it, like that something is an array at some point in time.

Reply:
Also: Style nit: We'd normally put an empty line before and after this `it` block

Reply:
But this is like a canary test; instead of getting some error in unexpected places, we ensure a certain API.